medicines/doc-index-updater/Makefile

```diff
@@ -10,6 +10,10 @@ default: ## Run locally
 test: ## Run tests [TEST=test_name (optional)]
 	export $$(cat .env .env.overrides 2> /dev/null | xargs) && cargo test $$TEST
 
+.PHONY: e2e
+e2e: ## Run end to end acceptance tests [TEST=test_name (optional)]
+	export $$(cat .env .env.overrides 2> /dev/null | xargs) && cargo test $$TEST -- --ignored
```

Comment:
Doesn't seem great overloading ignored tests as a organising tests. Could we split the e2e tests out into a separate crate instead?

Reply:
I agree but that wasn't part of this change - actually the indirection by having e2e target here allows us to change that.